geniza/corpus/management/commands/data_import.py

```diff
@@ -65,3 +66,23 @@ def import_libraries(self):
                 object_repr=str(library),
                 change_message=self.logentry_message,
                 action_flag=ADDITION)
+
+    def import_languages(self):
+        language_content_type = ContentType.objects.get_for_model(LanguageScript)
+
+        LanguageScript.objects.all().delete()
+        language_data = self.get_csv('languageScripts')
```

Comment:
Can we do language_scripts or just languages instead? (more pythonic)

Please add a placeholder to the sample local settings for this.